.github/CODEOWNERS

```
@@ -4,7 +4,7 @@
 *          @timneutkens @ijjk @shuding @huozhi
 /.github/  @timneutkens @ijjk @shuding @styfle @huozhi @padmaia
 /docs/     @timneutkens @ijjk @shuding @styfle @huozhi @padmaia @leerob @lfades @molebox
-/examples/ @timneutkens @ijjk @shuding @leerob @lfades
+/examples/ @timneutkens @ijjk @shuding @leerob @lfades @steventey
```

```suggestion
/examples/ @timneutkens @ijjk @shuding @leerob @lfades @steven-tey
```
Just double checking, is the dash not supposed to be here?